docs/source/features/packaging_output_models.md

```diff
@@ -54,9 +78,28 @@ Olive packaging configuration is configured in `PackagingConfig` in Engine confi
     * `type [PackagingType]`:
       Olive packaging type. Olive will package different artifacts based on `type`.
     * `name [str]`:
-      For `PackagingType.Zipfile` type, Olive will generate a ZIP file with `name` prefix: `<name>.zip`. By default, the output artifacts will be named as `OutputModels.zip`.
-    * `export_in_mlflow_format [bool]`:
-      Export model in mlflow format. This is `false` by default.
+      For `PackagingType.Zipfile` type, Olive will generate a ZIP file with `name` prefix: `<name>.zip`.
+      For `PackagingType.AzureMLModels` and `PackagingType.AzureMLData`, Olive will use this `name` for Azure ML resource.
+      The default value is `OutputModels`.
```

Comment:
Default value is `AzureMLModels`?

Reply:
This value will be displayed on AML as assets name. Showing `AzureMLModels` in AML Models looks weird to me, cause they are indeed AML models. This name more means output models from Olive.

Reply:
Okay, I got incorrect understanding, I am thinking the name somehow is PackagingType.